pkg/pillar/types/zedmanagertypes.go

```diff
@@ -169,6 +169,15 @@ type AppInstanceStatus struct {
 	ErrorAndTimeWithSource
 }
 
+type AppCount uint8
+type AppInstanceSummary struct {
+	UUIDandVersion UUIDandVersion
+	TotalApps      AppCount // Total number of apps configured
+	TotalRunning   AppCount // Total number of apps in running or booting state
+	TotalHalted    AppCount // Total number of apps in halted state
```

Comment:
I think we want to refine these a bit so that we count the number of apps intended to run (those with EffectiveActivate set) vs the number actually running (those with Activated set) and then compare for greater than (something is about to start), less than (something is in the process of halting) and equal (everything which should be running is running; everything which should be stopped is halted). So maybe the counters should be TotalStarting, TotalRunning, TotalStopping instead.

Reply:
ok

Reply:
Fixed